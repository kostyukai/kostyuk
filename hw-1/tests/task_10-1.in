A 12 32000 1.96
