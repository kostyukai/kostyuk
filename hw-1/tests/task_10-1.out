A 12 32000 1.960000
